src/services/stringCompletions.ts

```diff
@@ -388,7 +388,7 @@ function getStringLiteralCompletionEntries(sourceFile: SourceFile, node: StringL
                 // Get string literal completions from specialized signatures of the target
                 // i.e. declare function f(a: 'A');
                 // f("/*completion position*/")
-                return argumentInfo && getStringLiteralCompletionsFromSignature(argumentInfo.invocation, node, argumentInfo, typeChecker) || fromContextualType();
+                return argumentInfo && getStringLiteralCompletionsFromSignature(argumentInfo.invocation, node, argumentInfo, typeChecker) || fromContextualType(ContextFlags.None);

```

Comment:
`getStringLiteralCompletionsFromSignature` is already using the "inference blocking strategy" so it doesn't make sense (at least usually) for the fallback request~ to use it as well

Reply:
This is kinda using a simplified approach from the one that I was taking [here](https://github.com/microsoft/TypeScript/pull/53996/files#diff-3348dda4cea653e350948f038ce30f8c824e834bc7354128cc4c1d6858370084R391). So that other PR is also fixing this as is this PR: https://github.com/microsoft/TypeScript/pull/54121